board/forms.py

```diff
@@ -0,0 +1,35 @@
+from django import forms
+
+class PostForm(forms.Form):
+    """
+    A form representing the creation of a new post.
+    - `name` is the author's name
+    - `message` is the message being written in the post
+    - `photo` is the photo(s) attached to the post
+
```

Comment:
Following the Google style guides (https://google.github.io/styleguide/pyguide.html), the docstring for attributes should look like:

```
A form representing the creation of a new post.

Class Attributes
    name: the author's name
    message: the message being written in the post
```

Also note that in their example they specify that if the code does not use typing, types should be specified here!